examples/nivisa_dmm_measurement/_visa_dmm.py

```diff
@@ -0,0 +1,149 @@
+"""Custom instrument driver for MeasurementLink NI-VISA examples."""
+
+import logging
+import pathlib
+import sys
+from enum import Enum
+from types import TracebackType
+from typing import (
+    Optional,
+    Type,
+)
+
+import pyvisa
+import pyvisa.resources
+import pyvisa.typing
+
+
+if sys.version_info >= (3, 11):
+    from typing import Self
+else:
+    from typing_extensions import Self
+
+
+# Pin map instrument type constants for NI Instrument Simulator v2.0
+INSTRUMENT_TYPE_DMM_SIMULATOR = "DigitalMultimeterSimulator"
+
+SIMULATION_YAML_PATH = pathlib.Path(__file__).resolve().parent / "_visa_dmm_sim.yaml"
+"""
+Path to a simulation YAML file that can be used to simulate an NI Instrument Simulator v2.0
+with pyvisa-sim.
+"""
+
+RESOLUTION_DIGITS_TO_VALUE = {"3.5": 0.001, "4.5": 0.0001, "5.5": 1e-5, "6.5": 1e-6}
+
+
+class Function(Enum):
+    """Function that represents the measurement type."""
```

Comment:
No, it's not.

"Enum that represents the measurement function"?

Reply:
Updated it to `Enum that represents the measurement function`